pkg/btf/btf.go

```diff
@@ -109,9 +110,54 @@ func FindBTFStruct(name string) (*btf.Struct, error) {
 	}
 
 	err = spec.TypeByName(name, &ty)
+	if err != nil && errors.Is(err, btf.ErrMultipleMatches) {
+		logger.GetLogger().Warn("Multiple matches found in BTF spec; using first one", "struct", name)
+		// In case of multiple matches, fallback at using first available
+		err = firstTypeByName(spec, name, &ty)
```

Comment:
could we call it instead of spec.TypeByName ? IIUC like this we possibly do 2 searches